index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require 'src/db.php';
+require 'src/functions.php';
+
+$db = connectToDb('artistsApp');
+$artists = getAllArtists($db);
+
+
+?>
+
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Chloe's Collector App</title>
+    <link rel="stylesheet" href="https://use.typekit.net/rkk2hbc.css">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+    <html lang="en"></html>
+</head>
+    <header>
+        <h1 tabindex=0 >ARTISTS</h1>
+    </header>
```

Comment:
This should be within the body tag - all content visible on the page should live within the body tag.

Reply:
done🙂